src/main/java/edu/umich/ctools/sectionsUtilityTool/Friend.java

```diff
@@ -0,0 +1,317 @@
+			}
+			
+		}else {
+			M_log.error("File path for (sectionsToolPropsPathSecure.properties) is not provided");
+		}
+
+		Properties systemProps = System.getProperties();
+		String keyStoreType = (String) systemProps.get("javax.net.ssl.keyStoreType");
+		String trustStoreType = (String) systemProps.get("javax.net.ssl.trustStoreType");
+		if (keyStoreType != null && !KEYSTORETYPE_PKCS12.equals(keyStoreType)) // existing keyStoreType 
+		{
+			M_log.error(this + " init: existing settings of SSL keyStoreType mismatch: " + keyStoreType );
+			sslInitialized = false;
+		}
+		else if (trustStoreType != null  && !TRUSTSTORETYPE_JKS.equals(trustStoreType)) // existing trustStoreType
+		{
+			M_log.error(this + " init: existing settings of SSL trustStoretype mismatch: " + trustStoreType );
+			sslInitialized = false;
+		}
+		else
+		{	
+			// key store
+			systemProps.put("javax.net.ssl.keyStoreType", KEYSTORETYPE_PKCS12);
+			systemProps.put("javax.net.ssl.trustStoreType", TRUSTSTORETYPE_JKS);
+
+			if (ksFileName.isEmpty())
+			{
+				// log error for missing keystore file path
+				M_log.error(this + " init the ctools.friend.keystorefile path is not defined. ");
+			}
+			else
+			{
+				systemProps.put("javax.net.ssl.keyStore",ksFileName);
+			}
+
+			if (ksPwd.isEmpty())
+			{
+				// log error for missing keystore password
+				M_log.error(this + " init the ctools.friend.keystorepassword is not defined. ");
+			}
+			else
+			{
+				systemProps.put("javax.net.ssl.keyStorePassword",ksPwd);
+			}
+
+			// set system properties
+			System.setProperties(systemProps);
+
+			sslInitialized = true;
+		}
+	}	
+
+	/**
+	 * Returns to uninitialized state.
+	 */
+	public void destroy()
+	{
+		M_log.info(this +".destroy()");
+
+		if (sslInitialized)
+		{
+			// remove ssl system settings
+			// configure JSSE system properties: 
+			// http://fusesource.com/docs/broker/5.3/security/SSL-SysProps.html
+			Properties systemProps = System.getProperties();
+			//systemProps.remove("javax.net.debug");
+			//important: http://stackoverflow.com/questions/6680416/apache-cxf-exception-in-ssl-communication-sockettimeout
+			//java.lang.System.setProperty("sun.security.ssl.allowUnsafeRenegotiation", "true");
+			systemProps.remove("javax.net.ssl.keyStoreType");
+			systemProps.remove("javax.net.ssl.trustStoreType");
+			systemProps.remove("javax.net.ssl.keyStore");
+			systemProps.remove("javax.net.ssl.keyStorePassword");
+			System.setProperties(systemProps);
+
+			sslInitialized = false;
+		}
+	}	
+
+	/*
+	 * Check if an account exists
+	 * Return a single integer with the following (It all succeeds of fails)
+	 * @param email friend account Email
+	 * @return
+	 * -1 = bad e-mail address
+	 * 0 = no friend account
+	 * 1 = friend account exists 
+	 *
+	 */
+	public static int checkAccountExist(String email) {
+		int rv = 0;	// default to be "no friend account"
+		try {
+			Xclient = new XmlRpcClient();
```

Comment:
Could two threads be calling checkAccountExit at the same time?  If so then a static variable isn't safe.  Is Xclient used outside of the method?  If not then it doesn't need to be static.  If it is please make it clear that it is safe to use.  How about initializing in the init method?